doc/source/sbazv_de.md

```diff
@@ -0,0 +1,50 @@
+# Südbrandenburgischer Abfallzweckverband
+
+Support for schedules provided by [https://www.sbazv.de/](https://www.sbazv.de/).
+
+## Configuration via configuration.yaml
+
+```yaml
+waste_collection_schedule:
+  sources:
+    - name: sbazv_de
+      args:
+        city: CITY
+        district: DISTRICT
+        street: STREET
+```
+
+### Configuration Variables
+
+**city**<br>
+*(string) (required)*
+
+**district**<br>
+*(string) (required)*
+
+**street**<br>
+*(string) (required)*
+
+## Example
+
+```yaml
+waste_collection_schedule:
+  sources:
+    - name: sbazv_de
+      args:
+        city: "wildau"
+        district: "wildau"
+        street: "Ahornring"
+```
+
+## How to get the source arguments
+
+### Hardcore Variant: Extract arguments from website
+
+Another way get the source arguments is to extract the arguments from the website using a (desktop) browser with developer tools, e.g. Google Chrome:
```

Comment:
Please rephase. The text doesn't really match.
Esp:
- "Another way..." --> there is no other way described
- "5. ... You can see the string to download your ics file" --> How to extract the 3 arguments from the string?